### PR TITLE
Return CID in upload complete action

### PR DESCRIPTION
**Pull request by clostao**
Complete upload endpoint returned nothing (200 OK) we need the CID to be returned for use cases such as publish a file and automatically send a link to the file.

### CODE REVIEW

File: backend/src/controllers/upload.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import multer from "multer";
 import { FolderTreeFolderSchema } from "../models/objects/folderTree.js";
 import { uploadOptionsSchema } from "../models/uploads/upload.js";
 import { z } from "zod";
+import { cidToString } from "@autonomys/auto-drive";
```
Comment (marc-aurele-besner) [inline backend/src/controllers/upload.ts:8]: This line is useless
Furthermore, can you add workflow to run eslint to catch these things as part of our ci/cd ?

File: backend/src/useCases/uploads/uploadProcessing.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ import {
   uploadsRepository,
 } from "../../repositories/uploads/uploads.js";
 import { filePartsRepository } from "../../repositories/uploads/fileParts.js";
+import { CID } from "multiformats";
```
Comment (marc-aurele-besner) [inline backend/src/useCases/uploads/uploadProcessing.ts:24]: do we export this type in auto-sdk? if so we should export it, basically anything we need from multiformats should be exported in our package

**Review by marc-aurele-besner — CHANGES_REQUESTED** [review]
We should have workflows for both BE and FE to run build and lint

File: backend/src/useCases/uploads/uploadProcessing.ts (hunk 2 above)
Comment (clostao) [reply backend/src/useCases/uploads/uploadProcessing.ts:24]: No, they're not exported I'll add an issue on the sdk

**Comment by clostao** [comment]
> We should have workflows for both BE and FE to run build and lint

Yeah, I'll add that in a another PR

**Review by marc-aurele-besner — APPROVED** [review]

**Pull request merged**